tests/enrich/image/rekognition_test.php

```diff
@@ -22,6 +22,8 @@
  * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
  */
 
+namespace search_elastic\enrich\image;
```

Comment:
This namespace doesn't feel right

Reply:
It does look ok based on coding styles: 
When using namespaces, the namespace of the test class should match the namespace of the code under test.